Book-A-Desk.Api.Tests/FilteredBookingsTest.fs

```diff
@@ -0,0 +1,90 @@
+module Book_A_Desk.Api.Tests.FilteredBookingsTest
+
+open FsToolkit.ErrorHandling
+open System
+open Xunit
+
+open Book_A_Desk.Api.Models
+open Book_A_Desk.Domain.Office.Domain
+open Book_A_Desk.Infrastructure.DynamoDbEventStore
+open Book_A_Desk.Domain
+open Book_A_Desk.Infrastructure
+
+open Book_A_Desk.Domain.Events
+open Book_A_Desk.Domain.Reservation
+open Book_A_Desk.Domain.Reservation.Queries
+
+open Book_A_Desk.Domain.Reservation.Events
+
+
+let officeId = Guid.NewGuid ()
+let date = DateTime(2100,02,01)
+
+let createDeskBookedEvent date officeId email :DomainEvent =
+    ({
+            Date = date
+            EmailAddress = "anEmail" |> EmailAddress
+            OfficeId = officeId |> OfficeId
+    }: Events.DeskBooked) |> DeskBooked |> ReservationEvent
+let bookings: DomainEvent seq = 
+    [
+        createDeskBookedEvent date officeId "lasnikr"
+        createDeskBookedEvent date officeId "user"
+        createDeskBookedEvent date officeId "dummy"
+    ]
+        
+
+let mockProvideEventStore =
+        {
+            GetEvents = fun _ -> bookings |> Ok |> async.Return
+            AppendEvents = fun _ -> failwith "should not be called"
+        } : DynamoDbEventStore.DynamoDbEventStore
+
+let getBookingEvents : Option<seq<DomainEvent>> =
```

Comment:
This might be a misunderstanding of F#. This is not a function, it's a variable. The code here is actually run when the API.Tests project is loaded. If you want to turn it into a function, you need to add a `()`. So: `let getBookingEvents () = // the code to run`.

Although in this case, I don't think this code is useful. If you are writing an API Test, the event store should be injected in the `ApiDependencyFactory`, which means you don't need to call it. If you are writing a unit test, you should provide the list of bookings directly in the call to `ReservationsQueriesHandler.getFilteredBookings`, without going through a dependency.